emscripten.py

```diff
@@ -709,10 +709,13 @@ def keyfunc(other):
       asm_setup += 'var setTempRet0 = Runtime.setTempRet0, getTempRet0 = Runtime.getTempRet0;\n'
 
     if settings['BINARYEN']:
-      table_size = sum(map(lambda table: table.count(',') + 1, last_forwarded_json['Functions']['tables'].values()))
-      asm_setup += "\nModule['wasmTableSize'] = %d;\n" % table_size
+      def table_size(table):
+        s = table.count(',')
+        return s + 1 if s > 0 else 0
```

Comment:
er.. right. so `table` is actually the whole `var FUNCTION_TABLE_i = [...]` expression, so... yay parsing!